src/handlers/authentication.ts

```diff
@@ -8,19 +8,28 @@ import { logger, RainbowError } from '@/logger';
 
 const encryptor = new AesEncryptor();
 
-export async function getExistingPIN() {
+export async function getExistingPIN(): Promise<string | null> {
   try {
     const encryptedPin = await keychain.loadString(pinKey);
     // The user has a PIN already, we need to decrypt it
     if (encryptedPin) {
       const userPIN = await encryptor.decrypt(RAINBOW_MASTER_KEY, encryptedPin);
-      return userPIN;
+      return userPIN as string;
     }
     // eslint-disable-next-line no-empty
   } catch (e) {}
   return null;
 }
 
+export async function decryptPIN(encryptedPin: any) {
```

Comment:
did you mean to replace this `any` with `string | null` as well?

Reply:
I think I should

Reply:
I actually refactored this whole file in another PR stacked on top of this one
https://github.com/rainbow-me/rainbow/pull/4740/files#diff-21ac38fdbbad9f7aa84fbcb7135b490bf82c53b44db6354e06edc6c2610f7112R28